### ISSUE TITLE
`delete` does not throw if record is not found

### ISSUE THREAD

**Issue by leighman** [demonsters/prisma-mock#9]
Not sure if this is engine dependent but it should throw something like
```
        throw new PrismaClientKnownRequestError(
          'An operation failed because it depends on one or more records that were required but not found. Record to delete does not exist.',
          { code: '2025' }
        )
```
if you try to delete a record and at doesn't exist.

**Comment by stijnvanderlaan** [comment]
You are right, thanks for your feedback.

**Comment by stijnvanderlaan** [comment]
Fixed in v0.3.0
